src/components/SettingsModal.tsx

```diff
@@ -51,39 +56,60 @@ export const SettingsModal = ({ handleCloseModal, isModalOpen }: ISettingsModal)
 
   const { addAlert } = useAlert() || {};
 
+  const onChangeDsl = useCallback((value: string) => {
```

Comment:
Maybe in a separate one, but might be better to create a service and move this into it, like `FlowsService`